scripts/verify.py

```diff
@@ -66,7 +66,8 @@ def _main():
     sonde_info = yaml.load(open(args.sonde_info), Loader=yaml.SafeLoader)
 
     total_warnings = 0
-    for filename in tqdm.tqdm(args.infiles):
+    # for filename in tqdm.tqdm(args.infiles):
+    for filename in args.infiles:
```

Comment:
Is this a temporary fix which slipped in? Or should tqdm be removed / made optional?

Reply:
<facepalm> I try to keep my local environments small so made this change for my local testing, sorry. I reverted the change.

Reply:
Probably tqdm should be optional then... But that's for a separate PR. Thanks for reverting it.